example/ios/Podfile

```diff
@@ -10,80 +10,32 @@ project 'Runner', {
   'Release' => :release,
 }
 
-def parse_KV_file(file, separator='=')
-  file_abs_path = File.expand_path(file)
-  if !File.exists? file_abs_path
-    return [];
+def flutter_root
```

Comment:
iOS flutter has build failed problem when test. 
Those files are auto changed by removing the Podfile, Podfile.lock and running `flutter build ios`. 
https://stackoverflow.com/questions/65516334/flutter-error-invalid-podfile-file-no-implicit-conversion-of-nil-into-string